<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Pages</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="filesLayout">
        <item>
         <widget class="QTableWidget" name="listFile">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Save to</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QLineEdit" name="txtSaveSubFolder">
             <property name="text">
              <string>pdf_layout</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioSavePath">
             <property name="text">
              <string>Folder</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioSaveSubFolder">
             <property name="text">
              <string>SubFolder</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioSavePrefix">
             <property name="text">
              <string>Prefix (layout_)</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtSavePath">
             <property name="text">
              <string>D:\pdf_layout\</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioSaveSuffix">
             <property name="text">
              <string>Suffix (_NxM)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="controlLayout" stretch="0,0,0">
        <item>
         <widget class="QGroupBox" name="marginsGroupBox">
          <property name="title">
           <string>Margins</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QSpinBox" name="leftMargin"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="bottomLabel">
             <property name="text">
              <string>Bottom:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="rightMargin"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="topLabel">
             <property name="text">
              <string>Top:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="leftLabel">
             <property name="text">
              <string>Left:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="bottomMargin"/>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QCheckBox" name="borderCheckbox">
             <property name="text">
              <string>Draw border</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="rightLabel">
             <property name="text">
              <string>Right:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="topMargin"/>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="vSpace"/>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="hSpace"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="verticalGapLabel">
             <property name="text">
              <string>V-Space</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="4">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="horizontalGapLabel">
             <property name="text">
              <string>H-Space</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2" colspan="2">
            <widget class="QCheckBox" name="ratioCheckbox">
             <property name="text">
              <string>Keep ratio</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QCheckBox" name="dupletCheckbox">
             <property name="text">
              <string>Duplet</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QCheckBox" name="rotateCheckbox">
             <property name="text">
              <string>Rotate</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="pagePlacementGroupBox">
          <property name="title">
           <string>Page Placement and Scaling</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QSpinBox" name="gridColumns">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="pagesPerSheetLabel">
             <property name="text">
              <string>Pages per Sheet:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="gridRows">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="copies">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Full fill</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Columns</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rows</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Copies:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Direction</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="gridDirection">
             <item>
              <property name="text">
               <string>Left to Right</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Top to Bottom</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="pagesGroupBox">
          <property name="title">
           <string>Pages</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="pgStandardSize">
               <property name="text">
                <string>Standard</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="StandardSize">
               <property name="currentText">
                <string>A4</string>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>A5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A0</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="StandardOri">
               <item>
                <property name="text">
                 <string>Portrait</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Landscape</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="pg_custom">
             <item>
              <widget class="QRadioButton" name="pgCustomSize">
               <property name="text">
                <string>Custom:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="customSizeLayout">
               <item>
                <widget class="QSpinBox" name="customWidth">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="widthLabel">
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="customHeight">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>200</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="finishLayout">
        <item>
         <widget class="QPushButton" name="previewButton">
          <property name="text">
           <string>PREVIEW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="comboboxStyle">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string extracomment="Nothing">3x3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x5</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddStyle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveStyle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="okButton">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
